db/migrate/20230306105520_create_solidus_stripe_payment_intents.rb

```diff
@@ -1,9 +1,9 @@
 class CreateSolidusStripePaymentIntents < ActiveRecord::Migration[7.0]
   def change
     create_table :solidus_stripe_payment_intents do |t|
-      t.string :stripe_payment_intent_id
+      t.string :stripe_intent_id
```

Comment:
Does `null: false` & `unique: true` make sense here?

Reply:
I added it but then the specs were failing in weird ways, would be fine to open an issue targeting v5.1 or the next major?